health/health.d/backend.conf

```diff
@@ -1,3 +1,13 @@
+# Alert that backend will be disabled soon
+   alarm: backend_metrics_eol
+      on: netdata.backend_metrics
+   units: bytes
+    calc: sum($sent) + sum($buffered)
+   every: 5s
+    warn: $this != 0
+   delay: down 5m multiplier 1.5 max 1h
+    info: Backend is deprecated and it will be removed in a near future, migrate your configuration to exporting.conf
+      to: sysadmin
```

Comment:
i think you need availability alarm for it

like

https://github.com/netdata/netdata/blob/b3cfa54baa78ac90bccfde98adc14b193da9fd80/health/health.d/apache.conf#L4-L13

Reply:
Thank you for your suggestion @ilyam8 !
I updated it according our goals to inform users that backend should not be used more.